.clang-format

```diff
@@ -1,16 +1,26 @@
 ---
-BasedOnStyle: Google
-Language: Cpp
-Standard: Cpp03
+# Indentation for all files
+ColumnLimit: 120
+IndentWidth: 4
```

Comment:
@piponazo `IndendtWidth` is still specified to be 4, do you want to keep it?

Reply:
Hum ... I do not think 4 it is the default, I tried to remove it and reapply clang-format and it looks like the default is 2.

Anyways, some explicit values would not hurt :)

Reply:
I was confused because I suggested to keep the Google default which is 2 and you have a 👍 but didn't change it, that's why I commented.

I think 4 spaces is a lot for C++ but if you insist on that indent, I'd say we should def. go for 120 column limit and also modify the the negative indent of AccessModifierOffset to be -2

Reply:
ahhh I see your point now. Actually, lately I am also using `2` in other projects and I tend to prefer that indentation rather than `4`. 
Just to be sure, you are proposing to just delete the `IndentWidth` and keep `ColumnLimit:120' , right ?

Reply:
Yep 👍